### PR TITLE
Updated the French language

### PR DESCRIPTION
**Pull request by Ilovemyhous**
Some French sentences/words were updated, to have a better meaning.
Changed only `lang/po/Firmware_fr.po`.

### CODE REVIEW

**Comment by gudnimg** [comment]
It seems most of the files in the repo are changed for some reason.
image

**Comment by Ilovemyhous** [comment]
I'm trying to fix that ASAP, not sure why it edited everything.
Sorry for the inconvenience.

File: lang/po/Firmware_fr.po [hunk 1]
```diff
@@ -2359,8 +2360,8 @@ msgstr "Correct-X"
 
 #. MSG_XFLASH c=18
 #: ../../Firmware/ultralcd.cpp:5462
-msgid "XFLASH init"
-msgstr "Init XFLASH"
+msgid "XFLASH init."
+msgstr "Init. XFLASH"
```
Comment (gudnimg) [inline lang/po/Firmware_fr.po:2364]: @3d-gussner should we consider removing this string from translations? The string is essentially same in all current translations.

Comment (gudnimg) [reply lang/po/Firmware_fr.po:2364]: Actually on a second look, this string is no longer in the firmware 🤔 We need to re-sync the po files.

Comment (leptun) [reply lang/po/Firmware_fr.po:2364]: I removed it recently. It was from an unused test menu

**Comment by 3d-gussner** [comment]
@Ilovemyhous Thanks for the PR. Forwarded a review request to internal translators.

**Comment by Ilovemyhous** [comment]
See #4441.

**Pull request closed without merging**